### ISSUE TITLE
"hammr scan run" fails when searching scan on uforge

### ISSUE THREAD

**Issue by mikespub** [usharesoft/hammr#48]
Here's the error:
```
[root@webapp1 ~]# hammr -v
hammr version '0.2.5.8'
[root@webapp1 ~]# hammr scan run --ip 127.0.0.1 --scan-login root --name webapp1
INFO: no username nor password provided on command line, trying credentials file
INFO: Using credentials file: /root/.hammr/credentials.json
INFO: Using url https://uforge.usharesoft.com/api
Password for root@127.0.0.1:
... uforge-scan v2.55 (Nov 17 2015 16:25:44) (GIT Revision: 383245d)
... Distribution:        CentOS / 6.7 / i386
... Current System Name: Linux
...         Node Name:   webapp1
...         Release:     2.6.32-573.7.1.el6.i686
...         Version:     #1 SMP Tue Sep 22 19:05:58 UTC 2015
...         Machine:     i686
...         Domain:      (none)
... Server URL: https://uforge.usharesoft.com/api
... User: mikespub
... Testing connection to the service...
...                                     SUCCESS!
...
...
Searching scan on uforge ...
Traceback (most recent call last):
  File "/usr/lib/python2.6/site-packages/hammr/commands/scan/scan.py", line 101, in do_run
    myScannedInstance = self.api.Users(self.login).Scannedinstances.Get(None, Includescans="true", Name=doArgs.name)
TypeError: Get() got multiple values for keyword argument 'Includescans'
[root@webapp1 ~]#
```

**Comment by mikespub** [comment]
Forgot to add the probable reason why this failed :-)
```
[root@webapp1 ~]# hammr scan list
INFO: no username nor password provided on command line, trying credentials file
INFO: Using credentials file: /root/.hammr/credentials.json
INFO: Using url https://uforge.usharesoft.com/api
Getting scans for [mikespub] ...
+-----+--------------------------+--------+-----------------+
| Id  |           Name           | Status |  Distribution   |
+=====+==========================+========+=================+
| 767 | dbserver                 |        | CentOS 6 x86_64 |
+-----+--------------------------+--------+-----------------+
| 914 |         dbserver Scan #2 | 65%    |                 |
+-----+--------------------------+--------+-----------------+
| 768 | webapp1                  |        | CentOS 6 i386   |
+-----+--------------------------+--------+-----------------+
| 769 | webapp2                  |        | CentOS 5 i386   |
+-----+--------------------------+--------+-----------------+

Found 3 scans
```

**Comment by jbremond** [comment]
Dear Mikespub,

Thanks for this report. I will try to reproduced the error to understand the problem.

regards,
Joris

**Comment by alefebvr** [comment]
Hello,

We have reproduced the error. It seems to be due to a change in the backend UForge API which needs to be reflected onto hammr. We will work on it this week.

Best regards,
Alexandre Lefebvre, UShareSoft

**Comment by jbremond** [comment]
Hello @mikespub,

I have write the fix and it will be available in the next hammr release.

Regards,
Joris

**Comment by mikespub** [comment]
Thanks :-)

**Comment by segalaj** [comment]
This issue has been fixed by #55
